lib/tests/api/mod.rs

```diff
@@ -671,6 +671,139 @@ async fn delete_record_range() {
 	);
 }
 
+#[tokio::test]
+async fn changefeed() {
```

Comment:
By adding this, we ensure that the change feed work across the following backends:

- TiKV
- FDB
- SpeeDB
- RocksDB
- InMem